src/bridge_iroad/src/iroad_closedloop.cpp

```diff
@@ -91,32 +102,46 @@ void Teleop_IRoad::udp_cb(const ros::TimerEvent& event) {
 
   Matrix<double,3,1> r_f;       // Front vector in body frame
   r_f << 0.0,1.0,0.0;
-  steer = rad2deg(acos(r_tb.dot(r_f)/d_t));
+  
+  /*steer = rad2deg(acos(r_tb.dot(r_f)/d_t)); //debug: steering setting is equal to angle between vehicle heading and goal direction
 
-  steer = min(steer, 30.0);
+  steer = min(steer, max_steer);
```

Comment:
wouldn't this line be redundant given your definition later in the code?

Reply:
The line was only commented out and not deleted outright in case we wanted to restore the prior definition of the value for debugging purposes, but wouldn't run (and is superseded by the later definition that isn't commented out).